src/components/BaseModal/BaseModal.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import './BaseModal.css';
+import computeClassNames from '../../utils/compute-class-names';
+
+interface Props {
+  isClosed?: boolean;
+  renderHeader?: () => React.ReactNode;
+  onClose?: (value: boolean) => void;
+  children?: React.ReactNode;
+  className?: string;
+}
+
+function BaseModal({ isClosed = false, renderHeader, onClose, children, className = '' }: Props) {
+  const computedClassNames = computeClassNames({
+    'base-modal-container--open': isClosed,
+    'base-modal-container--closed': !isClosed
+  });
+
+  return (
+    <div className={`base-modal-container ${computedClassNames} ${className}`}>
+      <div
+        className="base-modal-container__backdrop"
+        onClick={() => onClose?.(!isClosed)}
+        role="button"
+        tabIndex={0}
+        onKeyDown={(e) => {
+          if (e.key === 'Enter') onClose?.(!isClosed);
```

Comment:
Cambiar la tecla a la Escape. 
```suggestion
          if (e.key === 'Enter') onClose?.(!isClosed);
```